NIAD-185: added section to walk-through integration testing using vsc…

michael-kainos opened the pull request:
Instructions added for using vscode rest client and docker containers connected to OpenTest

Commit:
NIAD-185: added section to walk-through integration testing using vscode rest client and docker containers connected to OpenTest

Alicja-Kainos commented on integration-tests/integration_tests/README.md:
```diff
@@ -60,6 +60,183 @@ source ./component-test-source.sh
+                "PARTY-KEY": "OpenTest Party Key Here",
+                "FILE-UPLOADED": "test file will go here",
+                "LOCAL_SCR_URL": "http://localhost:9000/",
+                "MESSAGE_ID": "6a6c8179-e8e3-419a-a6dc-0c07844ddef7"
```
I think it should be like this:
"BASE_URL": "http://localhost",
"INBOUND-PORT": "8082",
"OUTBOUND-PORT": "80",
"ROUTE-LOOKUP-PORT": "8088",
"FAKE-SPINE-PORT": "8091",
"ASID": "9XXXXXXXXXXX",
"PARTY-KEY": "A9XXXX-XXXXXXX",

GUID-idRoot, GUID-idAltRoot and MESSAGE_ID are generated with {{$guid}}.
LOCAL_SCR_URL is not used
SHORT-TIMESTAMP is predefined in requests
FILE-UPLOADED should not be in configuration, it is a String in one place of forward reliable request

michael-kainos replied:
Sure, will add this in.

Commit:
NIAD-185: added review changes

Commit:
NIAD-185: added override file
updated main compose file to reference rabbitmq image
removed local rabbitmq dockerfile
updated readme to reference created docker-compose files

Commit:
NIAD-185: added override file
updated main compose file to reference rabbitmq image
removed local rabbitmq dockerfile
updated readme to reference created docker-compose files

Commit:
NIAD-185: added override file
updated main compose file to reference rabbitmq image
removed local rabbitmq dockerfile
updated readme to reference created docker-compose files

Commit:
NIAD-185: added override file
updated main compose file to reference rabbitmq image
removed local rabbitmq dockerfile
updated readme to reference created docker-compose files

Commit:
Added RabbitMQ image docker release files

Commit:
NIAD-185: added outbound certificate variable to main docker-compose with default value
renamed override file for purpose
updated readme steps for override rename

Commit:
NIAD-185: added outbound certificate variable to main docker-compose with default value
renamed override file for purpose
updated readme steps for override rename

Commit:
NIAD-185: added outbound certificate variable to main docker-compose with default value
renamed override file for purpose
updated readme steps for override rename

Commit:
NIAD-185: added outbound certificate variable to main docker-compose with default value
renamed override file for purpose
updated readme steps for override rename

Commit:
NIAD-185: updated readme

Commit:
NIAD-185: removed rabbit mq duplicate line in override

Commit:
NIAD-185: reformatted doc
changed section for running docker scalable docker cluster

Commit:
NIAD-185: spell correct
